openslides_backend/action/user/set_password_self.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict
+
+from ...models.models import User
+from ...shared.exceptions import ActionException, PermissionDenied
+from ...shared.patterns import FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("user.set_password_self")
+class UserSetPasswordSelf(UpdateAction):
+    """
+    Action to update the own password.
+    """
+
+    model = User()
+    schema = DefaultSchema(User()).get_default_schema(
+        additional_required_fields={
+            "old_password": {"type": "string", "minLength": 1},
+            "new_password": {"type": "string", "minLength": 1},
+        }
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        if self.auth.is_anonymous(self.user_id):
+            raise PermissionDenied("Can't set password for anonymous")
```

Comment:
This is not a permission denied here but a ActionException (400 not 403).

Reply:
Ok, changed.